Automattic/WP-Job-Manager
# Display spinner when submitting initial job listing submission form

Also disables submit buttons to prevent double submit. Uses WordPress core's spinner for simplicity.

Fixes #1056

#### Changes proposed in this Pull Request:

* Show spinner and disable form submit buttons when submitting initial job listing form.

![small-submit-form-spinner 1](https://user-images.githubusercontent.com/68693/34790328-6390c258-f639-11e7-9377-0ab488a5fef8.gif)

#### Testing instructions:

* Submit job listing in frontend form and verify the submit button gets disabled and spinner is shown to the right of the submit button.
* Try with different themes. 
* Try with invalid fields and make sure you can submit once fixed.
* Try with different browsers.
